backend/db/resolve_fetch_failures_excess_items.py

```diff
@@ -12,13 +12,13 @@
 sys.path.insert(0, str(PROJECT_ROOT))
 from backend.db.utils import SCHEMA, fetch_status_set_success, get_db_connection, insert_from_dicts, \
     refresh_derived_tables, \
-    select_failed_fetches
+    reset_temp_refresh_tables, select_failed_fetches
 from enclave_wrangler.datasets import CSV_TRANSFORM_DIR, download_datasets
 from enclave_wrangler.utils import was_file_modified_within_threshold
 
 DESC = "Resolve failures due to too many expression items or members when fetching data from the Enclave's objects API."
 
-def resolve_fetch_failures_excess_items(use_local_db=False, cached_dataset_threshold_hours=0):
+def resolve_fetch_failures_excess_items(schema=SCHEMA, use_local_db=False, cached_dataset_threshold_hours=0):
```

Comment:
## `resolve_fetch_failures_excess_items` `schema`
Added missing param. This action is still broken though for a reason I don't remember, and apparently it doesn't need to run anyway. I don't know if we've ever had more than 1 such failure case, and it was on a TermHub test set I think.